rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -1048,15 +1047,13 @@ u32 SPUThread::get_ch_count(u32 ch)
 	case SPU_RdInMbox:        return ch_in_mbox.get_count();
 	case MFC_RdTagStat:       return ch_tag_stat.get_count();
 	case MFC_RdListStallStat: return ch_stall_stat.get_count();
-	case MFC_WrTagUpdate:     return ch_tag_upd == 0;
```

Comment:
I think you are wrong here. Because current implementation works as intended with request cancelling, and non-zero ch_tag_upd value means exactly this in emu implementation.